### PR TITLE
Lps 153770

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-153770

/cc @dantewang @tinatian

### CODE REVIEW

**Comment by dantewang** [comment]
Hi @zhangyan0701 could you review the CI results? Thanks!

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp [hunk 1]
```diff
@@ -236,7 +236,7 @@ long assetClassPK = DLAssetHelperUtil.getAssetClassPK(fileEntry, fileVersion);
 				<div class="sidebar-section">
 
 					<%
-					boolean isLatestVersion = fileVersion.equals(fileEntry.getLatestFileVersion());
```
Comment (tinatian) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp:239]: @slnn You should be able to use dlViewFileEntryDisplayContext.getFileVersion() here, inside this method, there is a cache.

**Comment by tinatian** [comment]
@slnn Please see my comment and resend.

Thanks.
Tina.

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp (hunk 1 above)
Comment (slnn) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp:239]: OK, got it

**Comment by slnn** [comment]
Resend in https://github.com/liferay-core-infra/liferay-portal/pull/794

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**